scripts/package.sh

```diff
@@ -30,7 +32,7 @@ rm -rf "${tmp_dir}"
 
 echo "Packaging"
 if [ "$goos" == "windows" ]; then
-  pwsh ${root_dir}/scripts/win_msi_build.ps1
+  powershell.exe -file "${root_dir}/scripts/win_msi_build.ps1" -arch amd64 -exporterName ${NAME} -version ${VERSION} -exporterGUID ${EXPORTER_GUID} -upgradeGUID ${UPGRADE_GUID} -licenseGUID ${LICENSE_GUID}
```

Comment:
Which part we were not able to port to Bash? cc/ @marcsanmi

Reply:
Honestly, I don't know. Maybe we could add a _sustainability_ task to dig into it again

Reply:
The bash script that was supposed to be the substitute for this one was in this [commit](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/137/commits/583fa3780598d24c56350ceda8abadc90adf2b1e). I checked it in a windows machine manually and it worked, the problem was while integrating it with the CI. I can't remember the exact job, but the error was similar to this [one](https://github.com/marcsanmi/newrelic-prometheus-exporters-packages/actions/runs/3684924562/jobs/6235259519) while testing the MSI generated.

I wouldn't state that It's not portable to bash, but as the goal of the task was to create and sign the MSI package and we were 1 day before the code freeze, we decided to merge it. However, I'd bet that if we spend some time we'll be able to find the culprit for that error and finally get rid of the PowerShell script.